packages/vm/src/runTx.ts

```diff
@@ -506,6 +506,11 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
     const keys = Object.keys(results.execResult.selfdestruct)
     for (const k of keys) {
       const address = new Address(hexToBytes(k))
+      if (this._common.isActivatedEIP(6780)) {
+        if (!results.execResult.createdAddresses![address.toString()]) {
```

Comment:
```suggestion
        if (results.execResult.createdAddresses![address.toString()] !== true) {
```
would this be better?

Reply:
Hmm - actually, if the address is not in the object, it would return false. (If the address is not created in this tx, and we want to selfdestruct it, this does not happen). So the check should be `=== undefined`. (Why does the linter not catch this? It should throw on non-boolean expressions?)